source/as-stream.ts

```diff
@@ -19,12 +24,12 @@ module.exports = options => {
 		};
 	}
 
-	const emitter = requestAsEventEmitter(options, input);
+	const emitter = requestAsEventEmitter(options, input) as RequestEmitter;
```

Comment:
Weirdly enough, I had to type cast this value, as opposed to simply typing it.
`new EventEmitter` from `node#events` seems to return a type of `internal` as opposed to return a `EventEmitter`. I'm a bit confused, it might be an issue with the node typings?

Reply:
Yup: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/events.d.ts

Can you check if there's an issue in https://github.com/DefinitelyTyped/DefinitelyTyped, and if not, open one?